### DISCUSSION TITLE
Re-mass auto request

### DISCUSSION THREAD

**Discussion by ChicoChubs** [sct/overseerr#2751] [Help]
Hello recently lost my radarr data somehow, I was wondering if I can auto sync with radarr all the request I have made, or should I start fresh again? 

Sorry about bad English 😆

**Comment by mboman** [comment]
I have the same issue. Re-requesting all requests that is is missing in Sonarr/Radarr is tedious. Likewise, the shows/movies that are already monitored in Sonarr/Radarr should be Overseerr's request log (as already approved).

**Comment by mduijm** [comment]
https://discord.com/invite/overseerr
